packages/@snowpack/plugin-svelte/README.md

```diff
@@ -9,21 +9,16 @@ npm install --save-dev @snowpack/plugin-svelte
 ```js
 // snowpack.config.json
 {
-  "plugins": [["@snowpack/plugin-svelte", { /* see "Plugin Options" below */}]]
+  "plugins": [
+    ["@snowpack/plugin-svelte", { /* see “Plugin Options” below */ }]
+  ]
 }
 ```
 
-#### Default Build Script
-
-```js
-{
-  // Matches all ".svelte" files
-  "scripts": {"build:svelte": "@snowpack/plugin-svelte"}
-}
-```
-
-You can override this by setting your own `"@snowpack/plugin-svelte"` build script.
-
 #### Plugin Options
 
-All plugin `options` are passed directly to the Svelte compiler. See [here](https://svelte.dev/docs#svelte_compile) for a list of supported options.
+| Name         |   Type    | Description                                                       |
+| :----------- | :-------: | :---------------------------------------------------------------- |
+| `sourceMaps` | `boolean` | Set to `false` to disable source map generation (default: `true`) |
```

Comment:
Since Svelte + Vue plugins now manage source maps themselves, you can now disable source maps in individual plugins if desired (they’re on by default)